news/_posts/2021-06-15-multistore-news-in-1.7.8.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "The multistore news in PrestaShop 1.7.8"
```

Comment:
I feel like this sentence could be improved a little. :) Here are a few suggestions:

"New multistore features in PrestaShop 1.7.8"
"Multistore in PrestaShop 1.7.8: What's new?"